### PR TITLE
Add procedure to attach EKS cluster to Kommander.

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

## Description of changes being made
The procedure describes how to attach an existing EKS cluster to Kommander.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+enterprise: false
+---
+
+You can attach existing Kubernetes clusters to Kommander. After attaching the cluster, you can use Kommander to examine and manage this cluster. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
+
+## Before you begin 
+
+This procedure requires the following items and configurations:
+
+- A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster
```
Comment (tbaums) [inline pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:17]: with administrative privileges.

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+1. Verify the kubeconfig file can access the EKS cluster.
+
+    ```bash
+    kubectl --kubeconfig $(pwd)/kommander-cluster-admin-config get all --all-namespaces
+    ```
+
+1. Copy `kommander-cluster-admin-config` to your clipboard.
+
+    ```bash
+    cat kommander-cluster-admin-config | pbcopy
```
Comment (tbaums) [inline pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:114]: this command will only work on MacOS. Need a linux version.

Comment (russurquhart1) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:114]: There is not a pbcopy for linux. Also, what is done with that file after it's on the clipboard?

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+layout: layout.pug
+navigationTitle: Attach Amazon EKS Cluster to Kommander
+title: Attach Amazon EKS Cluster to Kommander
+menuWeight: 7
+excerpt: Attach an existing EKS cluster to Kommander
+beta: true 
+enterprise: false
+---
+
+You can attach any existing Kubernetes clusters to Kommander, then use it to examine and manage those clusters later. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander.
```
Comment (cmastr) [inline pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:11]: If we're going to say you can examine and manage these later, we should link to the place to find info on examining and managing them.

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+You can attach any existing Kubernetes clusters to Kommander, then use it to examine and manage those clusters later. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
+
+## Before you begin 
+
+This procedure requires the following items and configurations:
+
+- A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster.
+- Konvoy v1.5.0-beta.0 or above, [installed and configured](https://docs.d2iq.com/ksphere/konvoy/1.5.0-beta/install/) for your Amazon EKS cluster, on your machine.
+- Kommander v1.0.1-beta.1 or above, [installed and configured](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/install/) on your machine.
```
Comment (cmastr) [inline pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:19]: This is a feature for 1.1, but it's in the 1.0 docs??

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+<p class="message--note"><strong>NOTE: </strong>This procedure assumes you have at least one existing and spun up Amazon EKS cluster. Refer to the Amazon EKS [website](https://aws.amazon.com/eks/) for setup and configuration information.</p>
+
+## Attach Amazon EKS Clusters to Kommander
+
+1. Ensure you are connected to your first EKS cluster. Enter the following commands:
```
Comment (cmastr) [inline pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:25]: how do they know which one is their "first" EKS culster?

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+      kind: ClusterRole
+      name: cluster-admin
+    subjects:
+    - kind: ServiceAccount
+      name: kommander-cluster-admin
+      namespace: kube-system
+    EOF
+    ```
+
+1. Create a kubeconfig file compatible with the Kommander UI. Enter these commands to set the following environment variables:
```
Comment (cmastr) [inline pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:63]: how do they know if it's compatible with the Kommander UI?

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md (hunk 3 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:11]: Yep, good catch!

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md (hunk 4 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:19]: Yes I put it in the wrong folder. I can just dupe this to the 1.1.0-beta equivalent folder, right? Or is it a separate brach like Konvoy is?

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md (hunk 6 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:63]: They really don't but this set of commands DOES create a compatible kubeconfig file.

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md (hunk 2 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:114]: Would they use xclip?

Comment (tbaums) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:114]: you could just have them run `cat kommander-cluster-admin-config` and copy/paste the yaml file.

you paste the file into the kommander ui. looks like it also supports uploading the file directly.

attaching a screenshot.
Screen Shot 2020-05-18 at 4 34 12 PM

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md (hunk 5 above)
Comment (tbaums) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:25]: good catch @cmastr . @russurquhart1 - should remove the word 'first'.

File: pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md (hunk 6 above)
Comment (tbaums) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:63]: correct. this whole rigamarole is due to the fact that the default kubeconfig file used/created by EKS is not compatible with the kommander UI.

Comment (cmastr) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:63]: Is there already a place where we describe how to create this file in the docs? If so, link those words to that topic. If not, we need to add a JIRA to document this rigamarole. :)

Comment (cmastr) [reply pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md:63]: ohhh! I see @russurquhart1 comment now. I'd probably reword to: 

Create a kubeconfig file according to the instructions below, which will make the file compatible with Kommander.

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+---
+
+<<<<<<< HEAD:pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md
+You can attach existing Kubernetes clusters to Kommander. After attaching the cluster, you can use Kommander to [examine and manage](https://docs.d2iq.com/ksphere/kommander/1.0/clusters/) this cluster. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
+
+## Before you begin 
+
+This procedure requires the following items and configurations:
+
+<<<<<<< HEAD:pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md
```
Comment (juliangieseke) [inline pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md:18]: This looks like Merge conflicts were commited

**Comment by cmastr** [comment]
closing in favor of #2951

**Pull request closed without merging**